compiler/test/compilable/testcstuff1.c

```diff
@@ -275,7 +275,7 @@ void test2()
     register int reg;
     const int ci;
     volatile int vi;
-    restrict int ri;
+//    restrict int ri;
```

Comment:
```suggestion
    int* restrict ri;
```
No need to comment it out, it can be made valid by putting `restrict` in the correct place.